blueprints/addon/files/.github/workflows/ci.yml

```diff
@@ -0,0 +1,65 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - [main, master]
+  pull_request: {}
+
+jobs:
+
+  lint:
+    name: "Lint"
+    env:
+      CI: true
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install node
+      uses: actions/setup-node@v2-beta
+      with:
+        node-version: 14.x
+
+    - uses: actions/cache@v2
+      with:
+        path: ${{ steps.cache.outputs.dir }}
+        key: ${{ runner.os }}-modules-${{ hashFiles('<%= yarn ? '**/yarn.lock' : '**/package-lock.json' %>') }}
+    - name: Install Dependencies
+      run: <%= yarn ? 'yarn' : 'npm' %> install
+    - name: Lint
+      run: <%= yarn ? 'yarn' : 'npm' %> lint
+
+  floating:
+    name: "Floating dependencies"
+    env:
+      CI: true
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        try-scenario: [
+          ember-lts-3.20,
+          ember-lts-3.24,
+          ember-release,
+          ember-beta,
+          ember-canary,
+          embroider-safe,
+          embroider-optimized
+        ]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install node
+      uses: actions/setup-node@v2-beta
+      with:
+        node-version: 14.x
+
+    - uses: actions/cache@v2
+      with:
+        path: ${{ steps.cache.outputs.dir }}
+        key: ${{ runner.os }}-modules-${{ hashFiles('<%= yarn ? '**/yarn.lock' : '**/package-lock.json' %>') }}
+    - name: Install Dependencies
+      run: <%= yarn ? 'yarn' : 'npm' %> install
+    - name: test
+      run: ./node_modules/.bin/ember try:one ${{ matrix.try-scenario }}
```

Comment:
this closely matches the existing travis setup.  This is probably a desirable start for the Actions setup.